lib/irb/lc/error.rb

```diff
@@ -13,8 +13,8 @@ def initialize(val)
     end
   end
   class NotImplementedError < StandardError
```

Comment:
I didn't know IRB has a `NotImplementedError` 🤯 

Should we remove this version and make the JA version simply inherit Ruby's `NotImplementedError`?

Reply:
I want to leave it to reduce difference between locale and possibility of locale dependent bug.